packages/react-composites/src/composites/CallComposite/types/CallControlOptions.ts

```diff
@@ -121,13 +110,37 @@ export interface CustomCallControlButtonCallbackArgs {
 /**
  * Response from {@link CustomCallControlButtonCallback}.
  *
- * Includes the props necessary to render a {@link  ControlBarButton} and indication of where to place the button.
+ * Includes the props necessary to render a {@link ControlBarButton} and a {@link DrawerMenuItem}
+ * with indication on where to place the button.
  *
  * @beta
  */
-export interface CustomCallControlButtonProps extends ControlBarButtonProps {
+export interface CustomCallControlButtonProps {
+  onItemClick?: () => void;
   /**
    * Where to place the custom button relative to other buttons.
    */
   placement: CustomCallControlButtonPlacement;
+  /**
+   * Whether the button is disabled
+   */
+  disabled?: boolean;
+  /**
+   * Icon to render
+   */
+  icon?: string;
```

Comment:
Should be `iconName`. Also should document how we use this name (looking at just this interface it is not clear how we'll get an icon from this string)

Reply:
Changed to iconName. Also changed how iconNames are provided. Icon must be of type CallWIthChatCompositeIcons or CallCompositeIcons.